python/tests/integration/arcticdb/version_store/test_basic_version_store.py

```diff
@@ -40,6 +40,14 @@
 from tests.util.date import DateRange
 
 
+if sys.platform == "win32":
+    # leave out Mongo as spinning up a Mongo instance in Windows CI is fiddly, and Mongo support is only
+    # currently required for Linux for internal use.
+    SMOKE_TEST_VERSION_STORES = ["lmdb_version_store", "s3_version_store"]  # SKIP_WIN
```

Comment:
Very minor: We don't really use the `smoke test` terminology anywhere in the code AFAIK. I'm probably just call this `TEST_VERSION_STORES`.

Reply:
I prefer it as is since _every_ version store in this file is really a "test version store".